docs/adoc/migration/MigrationGuide_23_2.adoc

```diff
@@ -94,4 +94,4 @@ To install the Scout SDK into your existing Eclipse IDE, use this P2 update site
 == Utility changes
 
 === DateUtility: New default date time format
```

Comment:
We use Title Case style -> DateUtility: New Default Date Time Format.

And I think there should be a comma before "are formatting the date identically.